python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1682,6 +1709,49 @@ def get_materialization_count_by_partition(
     ) -> Mapping[AssetKey, Mapping[str, int]]:
         return self._event_storage.get_materialization_count_by_partition(asset_keys, after_cursor)
 
+    # partitions storage
+
+    @traced
+    def get_mutable_partitions(self, partitions_def_name: str) -> Sequence[str]:
+        check.str_param(partitions_def_name, "partitions_def_name")
+        if self._partitions_storage is None:
+            check.failed(
+                "Your storage configuration currently does not support the partitions storage."
+            )
+        return self._partitions_storage.get_partitions(partitions_def_name)
+
+    @traced
+    def add_mutable_partitions(
```

Comment:
I've internally struggled on this name a little. On one hand, having `mutable` in there makes it clear that this only applies to `MutablePartitionsDefinition`s. `mutable_partition` is weird because it's not the partition itself that's mutable, it's the set of partitions.  We could name it something like `add_partition_to_mutable_partitions`, but that's a little wordy.

Curious if @prha or @OwenKephart have thoughts.